src/common/POFields/FieldAssignedTo/FieldAssignedTo.js

```diff
@@ -23,87 +26,85 @@ const columnMapping = {
 };
 const visibleColumns = ['name', 'patronGroup', 'username', 'barcode'];
 
-class FieldAssignedTo extends Component {
-  static propTypes = {
-    stripes: stripesShape.isRequired,
-    dispatch: PropTypes.func.isRequired,
-    change: PropTypes.func.isRequired,
-    assignedToValue: PropTypes.string,
-  };
+function FieldAssignedTo({ change, userId, mutator, stripes }) {
```

Comment:
Previously this component had a bug, name was displayed even if userId was cleared. So here kind of refactored version